ember.md

```diff
@@ -67,6 +68,36 @@ foo: undefined
 foo: null
 ```
 
+
+- #### 1.4 Multi-property Components
+
+	> Why? Better readability. 
+
+  + All components written in handlebars that have at least one property should be written in a multi-line format.
+
+  + Closing handlebars `}}` should be on a new line when writing in a multi-line format
+
+```Javascript
+// bad
+{{my-component CreamSoda='nope' Pepsi='nope'}}
+
+{{bryan-component theRightWay='nope'}}
+
+{{my-component
+	Sprite='nope' 
+	Fanta='nope'}}
+
+//good
+{{my-component
+	Coke='yes' 
+	SierraMist='yes'
+}}
+
+{{everyone-component
+	theRightWay='yes'
+}}
+```
+
```

Comment:
This is looking pretty good. I think my only suggestion now is the camelCase the property names.

Reply:
agreed, then done